### PR TITLE
added swager

### PR DESCRIPTION
**Pull request by VasylMoshun**

### CODE REVIEW

File: src/main/java/org/moshun/onlinebookstore/dto/userdto/UserRegistrationRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    @NotBlank(message = "firstName from 1 to 25 symbols")
+    @Size(min = 1, max = 25)
+    private String firstName;
+
+    @NotBlank(message = "userName from 1 to 25 symbols")
+    @Size(min = 1, max = 25)
+    private String lastName;
+
+    @NotBlank(message = "email from 8 to 25 symbols")
+    @Size(min = 8, max = 25)
```
Comment (Rommelua) [inline src/main/java/org/moshun/onlinebookstore/dto/userdto/UserRegistrationRequestDto.java]:
```suggestion
```

File: src/main/java/org/moshun/onlinebookstore/dto/userdto/UserRegistrationRequestDto.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    @Email
+    private String email;
+
+    @NotBlank(message = "password from 8 to 25 symbols")
+    @Size(min = 8, max = 100)
+    private String password;
+
+    @Size(min = 8, max = 100)
+    @NotBlank
```
Comment (Rommelua) [inline src/main/java/org/moshun/onlinebookstore/dto/userdto/UserRegistrationRequestDto.java]:
```suggestion
```

File: src/main/java/org/moshun/onlinebookstore/model/User.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    private String email;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false, unique = true, name = "first_name")
+    private String firstName;
+
+    @Column(nullable = false, unique = true, name = "last_name")
+    private String lastName;
```
Comment (Rommelua) [inline src/main/java/org/moshun/onlinebookstore/model/User.java]:
```suggestion
    @Column(nullable = false, name = "first_name")
    private String firstName;

    @Column(nullable = false, name = "last_name")
    private String lastName;
```

File: src/main/java/org/moshun/onlinebookstore/service/BookServiceImpl.java [hunk 4]
```diff
@@ -60,4 +65,20 @@ public List<Book> search(BookSearchParametersDto parameters, Pageable pageable)
         Specification<Book> bookSpecification = specificationBuilder.build(parameters);
         return bookRepository.findAll(bookSpecification,pageable).stream().toList();
     }
+
+    @Override
+    public UserResponseDto registration(UserRegistrationRequestDto requestDto)
```
Comment (Rommelua) [inline src/main/java/org/moshun/onlinebookstore/service/BookServiceImpl.java]: is it BookServiceImpl class?

**Review by koval-kn — APPROVED** [review]

**Pull request merged**